protocol/contracts/lease/src/api/mod.rs

```diff
@@ -26,8 +26,16 @@ pub type LpnCoinDTO = crate::finance::LpnCoinDTO;
 
 #[derive(Serialize, Deserialize, Clone)]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
-pub struct MigrateMsg {}
+pub struct MigrateMsg {
+    pub kind: MigrationKind,
+}
 
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub enum MigrationKind {
```

Comment:
Please add `// TODO` comment for removing it just so the IDEs can find it and we don't forget it. Probably not needed anywhere else because removing the definition will break the rest anyway.